### PR TITLE
Release 7.2

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description

V2X-Hub 7.2 release candidate branch for merge into master includes updated release notes.

## Motivation and Context
CARMA-Freight Testing V2X-Hub Release

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 V2X-Hub Release Notes
 ----------------------------
+Version 7.2, released April 12th, 2022
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.2 includes added functionality for the CARMACloud Plugin to support:
+- Receiving vehicle TCM acknowledgement messages and displaying appropriate positive/negative/no acknowledgement messages on V2X-Hub web UI.
```
Comment (snallamothu) [inline docs/Release_notes.md:8]: We need to include the issues for these enhancements below.

File: examples/tmx-exampleapps/SampleBSMPlugin/src/SampleBSMPlugin.cpp [hunk 2]
```diff
@@ -0,0 +1,144 @@
+#include "SampleBSMPlugin.h"
```
Comment (snallamothu) [inline examples/tmx-exampleapps/SampleBSMPlugin/src/SampleBSMPlugin.cpp:1]: This change was not included in the release notes.  I think this was added for VOICES project for simulation and may be beneficial for others to use for similar purposes.

**Review by snallamothu — CHANGES_REQUESTED** [review]

File: docs/Release_notes.md (hunk 1 above)
Comment (paulbourelly999) [reply docs/Release_notes.md]: Included new created issues #348 #349 #350 .

File: examples/tmx-exampleapps/SampleBSMPlugin/src/SampleBSMPlugin.cpp (hunk 2 above)
Comment (paulbourelly999) [reply examples/tmx-exampleapps/SampleBSMPlugin/src/SampleBSMPlugin.cpp:1]: Added a bullet for the Sample plugin changes

**Pull request merged**